.github/workflows/ci_test-tpu.yml

```diff
@@ -18,7 +18,12 @@ env:
 
 jobs:
   setup-build-publish-deploy:
-    name: tpu-testing-job
+    if: |
```

Comment:
here is not needed as this is triggered only on master

Reply:
okay